### PR TITLE
Source and sink doc

### PR DESCRIPTION
**Pull request by Shell32-Natsu**
Clarify source functions and sink functions are run by `kpt fn run`.

### CODE REVIEW

File: site/content/en/concepts/functions/_index.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ See [Configuration Functions Specification][spec] for further details.
 
 There are two special-case functions: source functions and sink functions.
 
+Note: Source functions and sink functions are *functions*. They are invoked by command `kpt fn run` instead of `kpt fn source` or `kpt fn sink`.
```
Comment (prachirp) [inline site/content/en/concepts/functions/_index.md:52]: Can you split this note into two? One note for source functions which goes at the bottom of the Source Function section and other note at bottom of Sink Function section. I think it's better to introduce the concept first and then give the clarification.

And let's add a question to the FAQ about this https://googlecontainertools.github.io/kpt/faq/

**Comment by Shell32-Natsu** [comment]
@prachirp Why there is a Travis CI build keeps in queued status?

**Comment by prachirp** [comment]
@Shell32-Natsu I'm not sure, I guess just wait for it to run.

**Comment by Shell32-Natsu** [comment]
The build on Travis CI has finished. These 2 checks shown actually point to the same build on Travis CI.

This PR(https://github.com/GoogleContainerTools/kpt/pull/700) has the same issue.

**Comment by Shell32-Natsu** [comment]
Will close this PR and create a new one

**Pull request closed without merging**